config/build.js

```diff
@@ -43,3 +47,40 @@ TYPES_TARGETS.forEach((target) => {
     }
   )
 })
+
+// Building providers
+
+const providersDir = path.join(process.cwd(), "/src/providers")
+
+const files = fs.readdirSync(providersDir, "utf8")
+
+let importLines = ""
+let exportLines = `export default {\n`
+files.forEach((file) => {
+  const provider = fs.readFileSync(path.join(providersDir, file), "utf8")
+  try {
+    // NOTE: If this fails, the default export probably wasn't a named function.
+    // Always use a named function as default export.
+    // Eg.: export default function YourProvider ...
```

Comment:
Love this! If something breaks and the contributor goes digging, comes across this, they will know exactly what went wrong and how to fix it!

Reply:
that's the goal. 😊